### PR TITLE
revert changes to image-preview

### PR DESCRIPTION
**Pull request by niwolf**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Team plentyShop about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

**Review by adumitrescu-plenty — DISMISSED** [review]

File: src/lib/terra-components.module.ts [hunk 1]
```diff
@@ -55,7 +55,8 @@ import { MatTreeModule } from '@angular/material/tree';
         MatPseudoCheckboxModule,
         MatToolbarModule,
         MatExpansionModule,
-        MatTreeModule
+        MatTreeModule,
+        MatPseudoCheckboxModule
```
Comment (pweyrich) [inline src/lib/terra-components.module.ts:59]: This is already added. See L55.
Didn't you revert using git?

Comment (niwolf) [reply src/lib/terra-components.module.ts:59]: I reverted the command `Merge pull request...` via GHD .

**Comment by pweyrich** [comment]
Reference: https://github.com/plentymarkets/terra-components/pull/1646/files

File: src/lib/terra-components.module.ts (hunk 1 above)
Comment (pweyrich) [reply src/lib/terra-components.module.ts:59]: Seems like reverting Merges that had conflicts is a mess 🙈

Comment (pweyrich) [reply src/lib/terra-components.module.ts:59]: but please remove this duplicate import

Comment (pweyrich) [reply src/lib/terra-components.module.ts:59]: and i think the expansion module import must be removed

Comment (niwolf) [reply src/lib/terra-components.module.ts]: done

Comment (niwolf) [reply src/lib/terra-components.module.ts]: Why the MatExpansionModule? That's not case from the image preview and maybe needed? in schematic maybe? 😅

**Review by pweyrich — APPROVED** [review]

File: src/lib/terra-components.module.ts (hunk 1 above)
Comment (pweyrich) [reply src/lib/terra-components.module.ts]: it is only used in the image preview

**Review by OliverBhm — APPROVED** [review]

**Pull request merged**